### ISSUE TITLE
Remote Ollama server

### ISSUE THREAD

**Issue by net-frog** [your-papa/obsidian-Smart2Brain#106]
### What happened?

Trying to use remote Ollama

### Error Statement

Access to fetch at 'http://192.168.100.102:11434/api/tags' from origin 'app://obsidian.md' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.

### Steps to Reproduce

1) install ubuntu with remote ollama
2) starting ollama on 0.0.0.0

### Smart Second Brain Version

1.0.2

### Debug Info

SYSTEM INFO:
	Obsidian version: v1.5.12
	Installer version: v1.5.12
	Operating system: Windows 10 Pro

**Comment by Leo310** [comment]
duplicate of https://github.com/your-papa/obsidian-Smart2Brain/issues/71. You need to set the OLLAMA_ORIGINS environment variable on your remote server before running Ollama.

**Comment by net-frog** [comment]
Thanks that resolved it.
